### PR TITLE
Fixes a runtime for ModPCs running Plexagon Access Management without a secondary card slot

### PR DESCRIPTION
**Pull request by zxaber**
## About The Pull Request
As said by the title.

Also fixes an oversight where the Plexagon HR Core app (job management) was reading worn ID rather than inserted ID.

Also adds names to differentiate between primary and secondary card readers so both show on the screwdriver remove menu.

Also makes using IDs on a modPC actually call the InsertID proc, so that having the secondary ID come first in the list of parts doesn't give it priority.

## Why It's Good For The Game
Runtimes are rude.

## Changelog
:cl:
fix: Fixed a runtime with Plexagon Access Management when the second ID slot is missing.
fix: Plexagon Access Management now follows the standard for ID reading and requires you to insert your card.
/:cl:

~~Since this computer takes so long to set up test servers I'm just gonna PR this now and fix the PR if the code isn't correct.~~ Works as intended

### CODE REVIEW

**Comment by Timberpoes** [comment]
Thanks for taking the time to iron out these lil' issues. I appreciate it!

Is this all tested and runtime free?

**Comment by zxaber** [comment]
Testing capacity is somewhat limited on this machine, but I did load up and go through every action that was having issues prior, with no trouble.

**Comment by Timberpoes** [comment]
> 
> 
> Testing capacity is somewhat limited on this machine, but I did load up and go through every action that was having issues prior, with no trouble.

Code changes look fairly straightforward. I can't see any issues with it. I'll swing back around and run this branch through my test setup that triggered the runtime after I've slept. If it all works out, I'll happily approve the changes with a view to merge at some point.

File: code/modules/modular_computers/computers/item/computer.dm [hunk 1]
```diff
@@ -350,6 +348,11 @@
 
 
 /obj/item/modular_computer/attackby(obj/item/W as obj, mob/user as mob)
+	// Check for ID first
+	if(istype(W, /obj/item/card/id))
+		if(InsertID(W))
+			return
```
Comment (bobbahbrown) [inline code/modules/modular_computers/computers/item/computer.dm:354]:
```suggestion
	if(istype(W, /obj/item/card/id) && InsertID(W))
		return
```

**Pull request merged**